### PR TITLE
Adding badges to README.md

### PR DESCRIPTION
**Pull request by sanchaez**
I've added a few badges to README.md. We can add more like download stats for SourceForge, CI status etc. etc.
Badges are simple icons. I used [shields.io](https://shields.io/) for the graphics.
New badges can be added using following syntax:
 ```[Badge Description](/clickable/link)```

### CODE REVIEW

**Comment by HBelusca** [comment]
In the shield.io list, I also see some Coverity status. I don't know if it's worth add it, yet we perform from time to time some Coverity analysis. I cc @AmineKhaldi so that he can say whether this could be useful.

**Comment by HBelusca** [comment]
I also see JIRA, SourceForge, and some other GitHub badges like eg. last commit.

**Comment by sanchaez** [comment]
Yes, shields.io is a service for creating badges. I don't know about possible integrations but they need to be explored indeed.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # ReactOS Project
 
-Current version: __0.4.6__
 ![ReactOS Logo](https://reactos.org/wiki/images/0/02/ReactOS_logo.png)
 
+[![Release 0.4.6](https://img.shields.io/badge/release-v0.4.6-green.svg)](https://sourceforge.net/projects/reactos/files/ReactOS/0.4.6/)
+[![ReactOS Build Environment](https://img.shields.io/badge/RosBE-v2.1.5-blue.svg)](https://sourceforge.net/projects/reactos/files/RosBE-Windows/i386/2.1.5/)
```
Comment (HeisSpiter) [inline README.md]: Don't forget we have two RosBE. One for Unix-like, one for Windows. I would put both or none (I'd rather go for the second one).

Comment (HBelusca) [reply README.md]: Having both of them would be nice actually. But then they should be mentioned separately.

Comment (sanchaez) [reply README.md]: @HBelusca I did that just now.

**Comment by AmineKhaldi** [comment]
Regarding Coverity here is [our badge](https://scan.coverity.com/projects/205/badge.svg?flat=1). I believe it's useful as it would hopefully motivate all of us to fix 'em all :)

**Comment by sanchaez** [comment]
Thanks @AmineKhaldi ! Added it to the [README](README.md).

**Comment by sanchaez** [comment]
Any other badges to add before merging?

**Comment by HBelusca** [comment]
We may try "Github all releases" or "Github Releases" (one should check the different versions of it), Github commits (?? if it can display the current git commit), Github commits since latest release (would correspond to the N in our git-describe "0.4.7-dev-N") : to be checked with e.g. @ColinFinck, and the "license" badge.

**Comment by sanchaez** [comment]
Hmmm now I see that shields.io badges are examples! I'll add those now.

**Comment by sanchaez** [comment]
@HBelusca  
_license_:  this doesn't work automatically as per #8 so I've added it manually.
_releases_: we don't have a release labeled "latest", and including previos releases there is pointless IMO.
_commits since latest release_: doesn't work as per above. But if you meant "since 0.4.7-dev", it works, but it displays 522 commits for me, which is wrong.
_current git commit_: it is already displayed at the top, no need for a badge.

**Comment by learn-more** [comment]
@sanchaez  Your first commit is with a hotmail address.

**Comment by sanchaez** [comment]
@learn-more will amend after it's ready.

**Pull request merged**